.github/workflows/prod-docker-mainnet.yml

```diff
@@ -0,0 +1,120 @@
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    name: Mainnet Production Pipeline
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 100
+
+      - uses: GoogleCloudPlatform/release-please-action@v3
+        id: release
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          release-type: node
+          package-name: ""
+          changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"ci","section":"CI / CD","hidden":false},{"type":"test","section":"Testing","hidden":false},{"type":"refactor","section":"Refactorings","hidden":false},{"type":"chore","section":"Miscellaneous","hidden":false}]'
+
+      - name: Set up Docker Buildx
+        id: buildx
+        # Use the action from the master, as we've seen some inconsistencies with @v1
+        # Issue: https://github.com/docker/build-push-action/issues/286
+        uses: docker/setup-buildx-action@master
+        # Only worked for us with this option on
+        with:
+          install: true
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          # Key is named differently to avoid collision
+          key: ${{ runner.os }}-${{ env.ENV }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-${{ env.ENV }}-buildx
+
+      - name: Log in to dockerhub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASS }}
+
+      - name: Extract metadata for docker
+        if: |
+          github.event_name == 'push' ||
+          startsWith(github.ref, 'refs/tags/v')
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: aeternity/mdw-frontend:mainnet # has to be changed
+          tags: |
+            type=raw,value=latest,enable=${{ endsWith(GitHub.ref, 'main') }}
+            type=raw,value=${{ steps.release.outputs.tag_name }},enable=${{ endsWith(GitHub.ref, 'main') }}
+            type=ref,event=tag
+            type=ref,event=pr
+
+      - name: Build and push docker image
+        if: |
+          github.event_name == 'push' ||
+          startsWith(github.ref, 'refs/tags/v')
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: nginx.Dockerfile
+          build-args: |
+            NUXT_APP_NODE_URL: https://mainnet.aeternity.io/v3
+            NUXT_APP_NODE_WS: wss://mainnet.aeternity.io/mdw/websocket
+            NUXT_APP_MDW_URL: https://mainnet.aeternity.io/mdw
+            NUXT_APP_OTHER_DEPLOYMENTS: TESTNET@https://explorer.testnet.aeternity.io
+            NUXT_APP_NETWORK_NAME: MAIN NET
+            NUXT_APP_ENABLE_FAUCET: false
+            NUXT_APP_FAUCET_API: https://testnet.faucet.aepps.com/account
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
```

Comment:
tag should be `master-mainnet` or `<sha>-mainnet`

Reply:
ye , something for clarifying mainnet on every release?